assets/js/blocks/stock-filter/edit.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+import { InspectorControls } from '@wordpress/block-editor';
```

Comment:
Suggesting to remove this empty line.

```suggestion
import { __ } from '@wordpress/i18n';
import { InspectorControls } from '@wordpress/block-editor';
```

Reply:
Fixed as you suggested!